packages/lab/src/multiline-input/MultilineInput.css

```diff
@@ -0,0 +1,230 @@
+
+  outline: var(--salt-focused-outlineWidth) var(--salt-focused-outlineStyle) var(--multilineInput-outlineColor);
+}
+
+/* Style applied if `readOnly={true}` */
+.saltMultilineInput.saltMultilineInput-readOnly {
+  --multilineInput-borderColor: var(--salt-editable-borderColor-readonly);
+  --multilineInput-borderStyle: var(--salt-editable-borderStyle-readonly);
+  --multilineInput-paddingLeft: 0;
+
+  background: var(--multilineInput-background-readonly);
+  cursor: var(--salt-editable-cursor-readonly);
+}
+
+/* Style applied if `readOnly={true}` and `fullBorder={true}` */
+.saltMultilineInput.saltMultilineInput-fullBorder.saltMultilineInput-readOnly {
+  --multilineInput-paddingLeft: var(--salt-spacing-100);
+}
+
+/* Styling when focused if `disabled={true}` or `readOnly={true}` */
+.saltMultilineInput-focused.saltMultilineInput-readOnly,
+.saltMultilineInput-focused.saltMultilineInput-disabled {
+  --multilineInput-borderWidth: var(--salt-size-border);
+  outline: none;
+}
+
+/* Style applied to selected text if `disabled={true}` */
+.saltMultilineInput-disabled .saltMultilineInput-textarea::selection {
+  background: none;
+}
+
+/* Style applied when `disabled={true}` */
+.saltMultilineInput.saltMultilineInput-disabled,
+.saltMultilineInput.saltMultilineInput-disabled:hover,
+.saltMultilineInput.saltMultilineInput-disabled:active {
+  --multilineInput-borderColor: var(--salt-editable-borderColor-disabled);
+  --multilineInput-borderStyle: var(--salt-editable-borderStyle-disabled);
+  --multilineInput-borderWidth: var(--salt-size-border);
+
+  background: var(--multilineInput-background-disabled);
+  cursor: var(--salt-editable-cursor-disabled);
+  color: var(--salt-text-primary-foreground-disabled);
+}
+
+/* Style for activation indicator */
+.saltMultilineInput-activationIndicator {
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  position: absolute;
+  border-bottom: var(--multilineInput-borderWidth) var(--multilineInput-borderStyle) var(--multilineInput-borderColor);
+}
+
+/* Style applied to start adornments */
+.saltMultilineInput-startAdornmentContainer {
+  align-items: center;
+  align-self: self-start;
+  display: inline-flex;
+  min-height: calc(var(--salt-size-base) + var(--salt-spacing-100));
+  padding-right: var(--salt-spacing-100);
+  column-gap: var(--salt-spacing-100);
+}
+
+/* Style applied to end adornments */
+.saltMultilineInput-endAdornmentContainer {
+  align-items: center;
+  align-self: self-end;
+  display: inline-flex;
+  min-height: calc(var(--salt-size-base) + var(--salt-spacing-100));
+  padding-left: var(--salt-spacing-100);
+  column-gap: var(--salt-spacing-100);
+}
+
+/* Style applied if `fullBorder={true}` */
+.saltMultilineInput-fullBorder {
+  --saltMultilineInput-border: var(--salt-size-border) var(--salt-container-borderStyle) var(--multilineInput-borderColor);
+  --saltMultilineInput-paddingLeft: var(--salt-spacing-100);
+  --saltMultilineInput-activationIndicator-border: none;
```

Comment:
i believe you need to override `border`. Currently it's `border: none;`

Reply:
ooh I deleted something on one commit and its broken. fixing now